docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -609,6 +609,30 @@ quarkus.oidc-client.credentials.jwt.signature-algorithm=RS512
 
 # The token endpoint URL is the default audience value, use the base address URL instead:
 quarkus.oidc-client.credentials.jwt.audience=${quarkus.oidc-client.auth-server-url}
+
+# custom subject instead of the client id :
+quarkus.oidc-client.credentials.jwt.subject=custom-subject
+
+# custom issuer instead of the client id :
+quarkus.oidc-client.credentials.jwt.issuer=custom-issuer
+----
+
+==== Apple POST JWT
+
+Apple OpenId Connect Provider uses a `client_secret_post` method where a secret is a JWT produced with a `private_key_jwt` authentication method but with Apple account specific issuer and subject properties.
+
+`quarkus-oidc-client` supports a non-standard `client_secret_post_jwt` authentication method which can be configured as follows:
+
+[source,properties]
+----
+quarkus.oidc-client.auth-server-url=${apple.url}
+quarkus.oidc-client.client-id=${apple.client-id}
+quarkus.oidc-client.credentials.client-secret.method=post-jwt
+
+quarkus.oidc-client.credentials.jwt.key-file=ecPrivateKey.pem
+quarkus.oidc-client.credentials.jwt.signature-algorithm=ES256
+quarkus.oidc-client.credentials.jwt.subject=${apple.subject}
+quarkus.oidc-client.credentials.jwt.issuer=${apple.issuer}
```

Comment:
This is nice, but optimistic and wrong, because other options are required:

```
quarkus.oidc.apple.auth-server-url=https://appleid.apple.com/
quarkus.oidc.apple.application-type=web-app
quarkus.oidc.apple.client-id=SECRET
quarkus.oidc.apple.credentials.jwt.key-file=SECRET
quarkus.oidc.apple.credentials.jwt.key-algorithm=ES256
quarkus.oidc.apple.credentials.jwt.token-key-id=SECRET
quarkus.oidc.apple.credentials.jwt.signature-algorithm=ES256
quarkus.oidc.apple.credentials.jwt.audience=https://appleid.apple.com
quarkus.oidc.apple.credentials.jwt.issuer=SECRET
quarkus.oidc.apple.credentials.jwt.subject=SECRET
quarkus.oidc.apple.authentication.scopes=openid,email,name
quarkus.oidc.apple.authentication.redirect-path=/Login/oidcLoginSuccess
quarkus.oidc.apple.authentication.extra-params.response_mode=form_post
quarkus.oidc.apple.authentication.force-redirect-https-scheme=true
quarkus.oidc.apple.credentials.client-secret.method=post_jwt
```